openmdao/recorders/base_recorder.py

```diff
@@ -117,6 +117,33 @@ def record_metadata_driver(self, recording_requester):
         """
         raise NotImplementedError()
 
+    def _get_metadata_system(self, recording_requester):
+        # Cannot handle PETScVector yet
+        from openmdao.api import PETScVector
+        if PETScVector and isinstance(recording_requester._outputs, PETScVector):
+            return None, None  # Cannot handle PETScVector yet
+
+        from six import iteritems
+        from openmdao.utils.options_dictionary import OptionsDictionary
```

Comment:
Seems like these imports can go to the top.

Reply:
Sorry. Had those there while writing it and missed those while cleaning up